### ISSUE TITLE
Error doing job: Task exception was never retrieved (None)

### ISSUE THREAD

**Issue by Bastian007** [nordicopen/easee_hass#470]
### The problem

HA log shows:
```
Logger: homeassistant
Source: custom_components/easee/controller.py:339
integration: Easee EV Charger ([documentation](https://github.com/nordicopen/easee_hass), [issues](https://github.com/nordicopen/easee_hass/issues))
First occurred: 21:06:29 (5 occurrences)
Last logged: 21:10:27

Error doing job: Task exception was never retrieved (None)

Traceback (most recent call last):
  File "/config/custom_components/easee/controller.py", line 706, in refresh_sites_state
    charger_data.check_latest_pulse()
  File "/config/custom_components/easee/controller.py", line 339, in check_latest_pulse
    elapsed = now - self.state["latestPulse"]
                    ~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pyeasee/utils.py", line 55, in __getitem__
    if type(self._storage[key]) == str and validate_iso8601(self._storage[key]):
            ~~~~~~~~~~~~~^^^^^
KeyError: 'latestPulse'
```

### Version of Easee integration having the issue?

0.9.59

### Version of Home Assistant Core having the issue?

2024.9.2

**Comment by olalid** [comment]
Same as https://github.com/nordicopen/easee_hass/issues/457

**Comment by olalid** [comment]
Probably solved in v0.9.60, please update and test.

**Comment by Bastian007** [comment]
I can confirm v0.9.60 solves the isssue. Thanks a lot!!!
